src/permutationtable.rs

```diff
@@ -26,7 +26,7 @@ const TABLE_SIZE: usize = 256;
 ///
 /// Table creation is expensive, so in most circumstances you'll only want to
 /// create one of these per generator.
-#[allow(missing_copy_implementations)]
+#[derive(Copy)]
```

Comment:
The reason I didn't derive `Copy` was because the table was so large.... maybe you are right about defaulting to references... I dunno. >_<